wsd/Admin.cpp

```diff
@@ -685,6 +691,35 @@ void Admin::pollingThread()
         LOG_TRC("Admin poll for " << timeout);
         poll(timeout); // continue with ms for admin, settings etc.
     }
+
+    if (!COOLWSD::IndirectionServerEnabled)
+        return;
+
+    _model.sendShutdownReceivedMsg();
+
+    const auto static closeMonitorMsgTimeout = std::chrono::microseconds(
+        COOLWSD::getConfigValue<int>("indirection_endpoint.migration_timeout_secs", 180) * 1000000);
```

Comment:
Why multiply with 1000000 and do the conversion to microseconds manually, when you can just as easily create `chrono::seconds`?
```
static const auto closeMonitorMsgTimeout = std::chrono::seconds(COOLWSD::getConfigValue<int>("indirection_endpoint.migration_timeout_secs", 180));
```

If you need it as microseconds:
```
static const std::chrono::microseconds closeMonitorMsgTimeout = std::chrono::seconds(COOLWSD::getConfigValue<int>("indirection_endpoint.migration_timeout_secs", 180));
```